### PR TITLE
feat(workflow): Fix "View in Discover" query

### PR DESCRIPTION
**Pull request by billyvg**
Changes columns to be issue.id, count(id), count_unique(user.id) and fixes sorting

### CODE REVIEW

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Comment by adhiraj** [comment]
👍

**Review by adhiraj — CHANGES_REQUESTED** [review]
Actually can you change the time period to match the period in the alert details graph?

File: src/sentry/static/sentry/app/views/alerts/details/body.tsx [hunk 1]
```diff
@@ -41,9 +45,12 @@ export default class DetailsBody extends React.Component<Props> {
     const discoverQuery: NewQuery = {
       id: undefined,
       name: (incident && incident.title) || '',
-      fields: ['title', 'user', 'last_seen'],
+      fields: ['issue.id', 'count(id)', 'count_unique(user.id)'],
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/alerts/details/body.tsx]: You can use the `issue` field now to get a shortid link instead of the autoincement.

**Review by markstory — APPROVED** [review]

**Review by adhiraj — APPROVED** [review]

**Pull request merged**